custom_model_runner/datarobot_drum/drum/runtime.py

```diff
@@ -88,5 +88,9 @@ def health():
     def predict():
         return {"message": "ERROR: {}".format(exc_value)}, HTTP_513_DRUM_PIPELINE_ERROR
 
+    @model_api.route("/transform/", methods=["POST"])
```

Comment:
@4dahalibut your original fix had this as `predict()`, let me know if this should be switched back